Makefile

```diff
@@ -14,18 +14,25 @@ help: ## Show this help message.
 	@egrep '^(.+)\:\ ##\ (.+)' ${MAKEFILE_LIST} | column -t -c 2 -s ':#'
 
 install: ## Install dependencies required to generate bindings & documentation
-install: install_dep vendorinstall
-	
+install: env version install_dep vendorinstall
+
+env:
```

Comment:
can we move this exports to install_prototool?

Reply:
Setting environment variables only affects the context of the makefile, right? Isn't there another way to do this in the makefile without the need to call export?

Reply:
resolving this conversation because I have made edits to the Makefile.